src/oci-umount.c

```diff
@@ -160,9 +159,9 @@ static int parse_mountinfo(struct mount_info **info, size_t *sz)
 	size_t len = 0;
 	int table_idx = 0;
 
-	fp = fopen(MOUNTINFO_PATH, "r");
+	fp = fopen("/proc/1/mountinfo", "r");
```

Comment:
Not from what I believe.  The /proc that is mounted is from the containers perspective.